Sources/ArcGISToolkit/FloorFilter/SiteFacilityTableViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+import Foundation
+
+/// Custom cell for site/facility.
+class SiteFacilityTableViewCell: UITableViewCell {
+    @IBOutlet var siteFacilityNameLabel: UILabel!
+    @IBOutlet var siteFacilityDotImg: UIImageView!
+}
```

Comment:
Does this need to be a custom table view cell or could it just be a cell with the default style? It seems like `siteFacilityNameLabel` is just `UITableViewCell.textLabel` and `siteFacilityDotImg` is just `UITableViewCell.imageView`.

Reply:
Thanks for pointing that out. Initially, this was a custom table view cell because of some of the other UI elements. I'd like to get this changed to a default style cell as an enhancement. I have logged an issue in the toolkit repo to fix it. https://devtopia.esri.com/runtime/common-toolkit/issues/166